tbx/project_styleguide/templates/patterns/pages/work/work_page.html

```diff
@@ -0,0 +1,108 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block meta_tags %}
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        window.dataLayer.push({
+            'author-title' : '{{ page.authors.all.0.author.role }}',
+            'author-name' : '{{ page.authors.all.0.author.name }}',
+            'published' : '{{ page.first_published_at|date:"Ymd" }}',
+            'tags' : [{% for tag in page.related_services.all %}'{{tag.name}}'{% if not forloop.last %}, {% endif %}{% endfor %}],
+            'read-time' : '{{ page.read_time }}',
+            'id': '{{ page.id }}',
+            'page-h1' : '{{ page.title }}',
+            'page-title' : '{{ page.title }} | Torchbox',
+            'page-path': '{{ page.slug }}',
+            'event': 'page-information'
+        });
+    </script>
+    <meta name="author" content="{{ page.authors.all.0.author.name }}">
+    <meta property="og:article:published_time" content="{{ page.date|date:'Y-m-d' }}" />
+    {{ block.super }}
+
+{% endblock %}
+
+{% block content %}
+    {% pageurl page.work_index as work_index_url %}
+
+    <div class="authors">
+        {% for page_author in page.authors.all %}
+            {% include "patterns/molecules/author/author.html" with author=page_author.author item=page %}
+        {% endfor %}
+    </div>
+
+    <div class="streamfield__title">
+        <h1 class="heading heading--one">{{ page.title }}</h1>
+
+        <div class="tags tags--with-title">
+            <span class="sr-only">Filter case studies by category</span>
+            {% for tag in page.services %}
+                {% include "patterns/atoms/tag/tag.html" with tag_slug=tag.slug tag_name=tag.name tag_link_base=work_index_url %}
```

Comment:
If we always want to target the parent page, this should work. We might even be able to ditch `WorkPage.work_index` and `work_index_url` too, perhaps?

```suggestion
                {% include "patterns/atoms/tag/tag.html" with tag_slug=tag.slug tag_name=tag.name tag_link_base='../' %}
```